.circleci/config.yml

```diff
@@ -76,11 +71,17 @@ jobs:
     working_directory: *working_dir
     steps:
       - attach_workspace: *attach_options
-      - restore_cache: *package_lock_key
+      - restore_cache:
+          name: Restore Yarn Package Cache
+          keys:
+            - raiden-v1-yarn-{{ checksum "yarn.lock" }}
```

Comment:
The old `cache_key` had a `.Branch` in it as well. might make sense to keep that.

Reply:
IMO, there's no point into caching different branches differently. Even because usually different PRs would be seen as different branches. If the `yarn.lock` matches (and actually even if don't), we want the yarn cache to be... _cached_, so it speeds up the install for new runs. Not sure why `.Branch` was there before, but I fail to see a reason for that.